Backend/Interview.Backend/RoomReviews/RoomReviewController.cs

```diff
@@ -93,4 +112,23 @@ public Task<RoomReviewDetail> Update([FromRoute] Guid id, [FromBody] RoomReviewU
     {
         return _roomReviewService.UpdateAsync(id, request, HttpContext.RequestAborted);
     }
+
+    /// <summary>
+    /// completion a review.
+    /// </summary>
+    /// <param name="request">User Request.</param>
+    /// <param name="currentUserAccessor">Current user accessor</param>
+    /// <returns>HTTP result</returns>
+    [Authorize]
+    [HttpPost("/complete")]
+    [Produces(MediaTypeNames.Application.Json)]
+    [ProducesResponseType(typeof(RoomReviewDetail), StatusCodes.Status201Created)]
+    public async Task<ActionResult> CompleteAsync([FromBody] RoomReviewCompletionRequest request, [FromServices] ICurrentUserAccessor currentUserAccessor)
+    {
+        var userId = currentUserAccessor.GetUserIdOrThrow();
+
+        await _roomReviewService.CompleteAsync(request, userId, HttpContext.RequestAborted);
+
+        return ServiceResult.Ok().ToActionResult();
```

Comment:
ServiceResult.Ok = 200 http code, not 201. Use Created method

Reply:
Sory, but it is complete method and i did not create something entity.

Reply:
I did method PUT for complete, because I don't create entity. I only update.